src/Microsoft.OData.Core/UriParser/Visitors/ISyntacticTreeVisitor.cs

```diff
@@ -145,19 +145,26 @@ public interface ISyntacticTreeVisitor<T>
         T Visit(FunctionParameterToken tokenIn);
 
         /// <summary>
-        /// Visits a AggregateToken
+        /// Visits a AggregateTransformationToken
         /// </summary>
-        /// <param name="tokenIn">The AggregateToken to bind</param>
-        /// <returns>A T node bound to this AggregateToken</returns>
+        /// <param name="tokenIn">The AggregateTransformationToken to bind</param>
+        /// <returns>A T node bound to this AggregateTransformationToken</returns>
         T Visit(AggregateToken tokenIn);
 
         /// <summary>
-        /// Visits a AggregateExpressionToken
+        /// Visits a AggregateToken
         /// </summary>
-        /// <param name="tokenIn">The AggregateExpressionToken to bind</param>
-        /// <returns>A T node bound to this AggregateExpressionToken</returns>
+        /// <param name="tokenIn">The AggregateTokenBase to bind</param>
+        /// <returns>A T node bound to this AggregateTokenBase</returns>
         T Visit(AggregateExpressionToken tokenIn);

```

Comment:
>AggregateExpressionToken [](start = 16, length = 24)

Although the <param> says this takes an AggregateTokenBase, the function signature still takes an AggregateExpressionToken.